Content.Client/EntityJobInfo/EntityJobInfoOverlay.cs

```diff
@@ -0,0 +1,146 @@
+using Content.Shared.Humanoid;
+using Content.Shared.Inventory;
+using Content.Shared.PDA;
+using Robust.Client.GameObjects;
+using Robust.Client.Graphics;
+using Robust.Shared.Enums;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+using Content.Shared.Access.Components;
+using Content.Shared.Roles;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Content.Client.EntityJobInfo;
+
+public sealed class EntityJobInfoOverlay : Overlay
+{
+    private readonly IEntityManager _entManager;
+    private readonly SharedTransformSystem _transform;
+    private readonly IPrototypeManager _prototypeManager;
+    private readonly InventorySystem _inventorySystem;
+    private readonly ShaderInstance _shader;
+    public override OverlaySpace Space => OverlaySpace.WorldSpaceBelowFOV;
+
+    public EntityJobInfoOverlay(IEntityManager entManager, IPrototypeManager protoManager, InventorySystem inventorySystem)
```

Comment:
This is still hardcoded for jobs when it should be generic for status indicators and in a separate PR.